modules/bar/notifications/index.ts

```diff
@@ -49,6 +55,9 @@ export const Notifications = (): BarBoxChild => {
                         });
 
                         if (showTotal) {
+                            if (hideCountForZero && notif.length === 0) {
```

Comment:
```suggestion
                            if (hideCountForZero && filteredNotifications.length === 0) {
```
Also overlooked that we support notifications filters. So we want to check based on the total count reflective of our filtered notifications. Otherwise you might have some notifications blocked by the filter and see 0 notifications, but the label isn't hidden since we're checking the unfiltered count.

Other than that this is ready to merge :)